src/attributes.cpp

```diff
@@ -2172,7 +2172,7 @@ namespace attributes {
 
                 const std::vector<Argument>& args = function.arguments();
                 for (std::size_t i = 0; i<args.size(); i++) {
-                    ostr() << "Rcpp::wrap(" << args[i].name() << ")";
+                    ostr() << "Shield<SEXP>(Rcpp::wrap(" << args[i].name() << "))";
```

Comment:
We could make this Rcpp::Shield for consistency with Rcpp::wrap but I see other identifiers without the namespace qualifier.